docker/dev/docker-compose.tests.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  db:
+    image: mariadb:latest
```

Comment:
latest? Shouldn't we target a bit more specific version?
If we are targeting the DEB packages for some specific Ubuntu/Debian release we might better align the DB version to the one provided by the official repo.
Maybe @elacunza can clarify about this.

Reply:
Fixed in the new PR #572

I'm using the same version proposed in production docker-compose.yml. Maybe we should review it.